controller_manager/src/ros2_control_node.cpp

```diff
@@ -18,11 +18,18 @@
 #include <string>
 #include <thread>
 
+#include "controller_manager/control_loop.hpp"
 #include "controller_manager/controller_manager.hpp"
 #include "rclcpp/rclcpp.hpp"
 
 using namespace std::chrono_literals;
 
+void sleep_until(rclcpp::Time time)
```

Comment:
It is because it enables testing this functionality without ever putting a sleep in the test. You should read the tests to understand why this is useful.

Reply:
You are right that it isn't needed as I also wrote this as a capture below :facepalm: